inference-engine/src/mkldnn_plugin/bf16transformer.h

```diff
@@ -18,7 +18,7 @@ class BF16Transformer {
     const InferenceEngine::details::caseless_set<std::string> _complementbf16 =
         { "relu", "tanh", "elu", "square", "abs", "sqrt", "linear", "bounded_relu", "soft_relu", "normalize",
           "sigmoid", "ReLU6", "not", "activation", "HSwish", "mish", "logistic", "mod", "resample",
-          "exp", "gelu", "clamp", "swish", "prelu", "pooling", "norm", "gather", "memory", "mvn", "crop", "activation",
+          "exp", "gelu", "clamp", "swish", "prelu", "pooling", "norm", "gather", "gather_elements", "memory", "mvn", "crop", "activation",
```

Comment:
Since I have enabled GatherElements for BF16 we need to cover it by CPU specific test. See examples here: https://github.com/openvinotoolkit/openvino/tree/master/inference-engine/tests/functional/plugin/cpu/single_layer_tests

Reply:
Added